src/App.jsx

```diff
@@ -3,31 +3,24 @@ import reactLogo from './assets/react.svg'
 import viteLogo from '/vite.svg'
 import './App.css'
 
-function App() {
-  const [count, setCount] = useState(0)
 
+const todoList=[
+  { id:0 , title:"Study the  React Book"},
+  { id:1 , title:"Watch Lesson's Video"},
+  { id:2 , title:"Participate Mentor group meeting"},
+  { id:3 , title:"Submit the assignment"}];
+
+function App() {
+ 
   return (
     <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <h1>Todo List</h1>
+      <ul >
+        {/* Map over the todoList array */}
+      {todoList.map(function(item){
```

Comment:
The following code snippet is preferrable:
```
{todoList.map(({ id, title }) => (
  <li key={id}>{title}</li>
))}
```
Here's what has been changed:

1. Replaced the regular function with an arrow function.
2. Used object destructuring to directly extract id and title from item.

the code is much clearer .